src/translate.rs

```diff
@@ -1003,6 +1003,88 @@ where
     }
 }
 
+pub struct PtrArray(*mut glib_sys::GPtrArray);
```

Comment:
This type is not used anywhere, is it? Should be removed then :)

Reply:
I mean, make it not public if it's only used internally here

Reply:
Looks like it needs to be public:
```
error[E0446]: private type `translate::PtrArray` in public interface
    --> /home/majorz/resin/libnm-rs-build/glib/src/translate.rs:1020:5
     |
1006 |   struct PtrArray(*mut glib_sys::GPtrArray);
     |   - `translate::PtrArray` declared as private
...
1020 | /     type Storage = (
1021 | |         Option<PtrArray>,
1022 | |         Vec<Stash<'a, <T as GlibPtrDefault>::GlibType, T>>,
1023 | |     );
     | |______^ can't leak private type
```
It is the same for the structs `SList` and `List`.

Reply:
Indeed. That sucks but what can we do :)